docs/realm-configuration.md

```diff
@@ -91,10 +91,9 @@ A list of custom key/values can be added at the end.
 | groupsMaxOutputSize                 | 100 |                                                                                     yes |        fr.insee.sugoi.groups.maxoutputsize | The maximum number of grouos outputs allowed                                                                                                                                                                               |
 | applicationsMaxOutputSize           | 100 |                                                                                     yes |  fr.insee.sugoi.applications.maxoutputsize | The maximum number of applications outputs allowed                                                                                                                                                                         |  
 | organizationsMaxOutputSize          | 100 |                                                                                     yes | fr.insee.sugoi.organizations.maxoutputsize | The maximum number of organizations outputs allowed                                                                                                                                                                        |  
+#### Userstorage configuration properties on password
```

Comment:
move it down in ## UserStorage configuration and don't forget to update the table of content at the top of this file